hack/aks/property-based-scheduling.md

```diff
@@ -0,0 +1,428 @@
+export PROPERTY_PROVIDER=aks
+
+declare -a MEMBER_CLUSTERS=($MEMBER_CLUSTER_1 $MEMBER_CLUSTER_2 $MEMBER_CLUSTER_3)
+
+# For each member cluster, create a service account in the hub cluster, and retrieve
+# the service account tokens.
+for i in "${MEMBER_CLUSTERS[@]}"
+do
+    kubectl create serviceaccount fleet-member-agent-$i -n fleet-system
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: fleet-member-agent-$i-sa
+        namespace: fleet-system
+        annotations:
+            kubernetes.io/service-account.name: fleet-member-agent-$i
+    type: kubernetes.io/service-account-token
+EOF
+done
+
+# Load the service account tokens to the respective member clusters.
+for i in "${MEMBER_CLUSTERS[@]}"
+do
+    kubectl config use-context $HUB_CLUSTER-admin
+    TOKEN=$(kubectl get secret fleet-member-agent-$i-sa -n fleet-system -o jsonpath='{.data.token}' | base64 -d)
+    kubectl config use-context "$i-admin"
+    kubectl delete secret hub-kubeconfig-secret --ignore-not-found
+    kubectl create secret generic hub-kubeconfig-secret --from-literal=token=$TOKEN
+done
+
+# Install the member agent.
+for (( i=0; i<3; i++ ));
+do
+    kubectl config use-context "${MEMBER_CLUSTERS[$i]}-admin"
+    helm install member-agent charts/member-agent/ \
+        --set config.hubURL=$HUB_SERVER_ADDR \
+        --set image.repository=$REGISTRY/member-agent \
+        --set image.tag=$TAG \
+        --set refreshtoken.repository=$REGISTRY/refresh-token \
+        --set refreshtoken.tag=$TAG \
+        --set image.pullPolicy=Always \
+        --set refreshtoken.pullPolicy=Always \
+        --set config.memberClusterName="${MEMBER_CLUSTERS[$i]}" \
+        --set logVerbosity=5 \
+        --set namespace=fleet-system \
+        --set enableV1Alpha1APIs=false \
+        --set enableV1Beta1APIs=true \
+        --set propertyProvider=$PROPERTY_PROVIDER
+done
+```
+
+After completing the commands, pick a cluster and verify that the member agent pod is up and running:
+
+```sh
+kubectl get pods -n fleet-system
+```
+
+You should see that the agent is in the ready state. Repeat the command on the other two member clusters; agents there should be ready as well.
+
+At last, join the member clusters to the hub cluster:
+
+```sh
+kubectl config use-context $HUB_CLUSTER-admin
+
+for i in "${MEMBER_CLUSTERS[@]}"
+do
+    cat <<EOF | kubectl apply -f -
+    apiVersion: cluster.kubernetes-fleet.io/v1beta1
+    kind: MemberCluster
+    metadata:
+        name: $i
+    spec:
+        identity:
+            name: fleet-member-agent-$i
+            kind: ServiceAccount
+            namespace: fleet-system
+            apiGroup: ""
+EOF
+done
+```
+
+You can verify the membership status using the command below; after a few moments all member clusters should have their `JOINED` status set to `True`:
+
+```sh
+kubectl get membercluster
+```
+
+## Experience the new property-based scheduling features
```

Comment:
Change this section heading to: "Try out Fleet property-based scheduling (preview)"